lib/litmus/type/string.ex

```diff
@@ -39,6 +39,8 @@ defmodule Litmus.Type.String do
     * `:trim` - Removes additional whitespace at the front and end of a string.
       Allowed values are `true` and `false`. The default is `false`.
 
+    * `:included` - Specifies a option list.
```

Comment:
I like this feature but I think I'd prefer having the name option be a list called `:valid` since it specifies the list of valid strings. This would make the using this feature like this:

```elixir
schema = %{
  "casing" => %Litmus.Type.String{
    included: ["lowercase", "uppercase"]
  }
}
```

Reply:
@dmlittle i think you tpyo'd `included` instead of `valid` 😆